airflow/cli/commands/internal_api_command.py

```diff
@@ -148,60 +139,54 @@ def kill_proc(signum, _):
                 gunicorn_master_proc.kill()
             sys.exit(0)
 
-        def monitor_gunicorn(gunicorn_master_pid: int):
+        def monitor_gunicorn(gunicorn_master_proc: psutil.Process | subprocess.Popen):
```

Comment:
I remember this psutikk/subprocess dualism hit us before. Nice to have it typed here, bit it's not typed in kill_proc - I do not know by heart if all teh methods are supported in both ? Can you add type hint there please?

Reply:
add typing; turns out there was some error in code that I fixed by copying&pasting the same section from webserver_command.py (BTW these 2 files have a lot of duplication so could use more refactoring)